common-methods.go

```diff
@@ -114,7 +115,20 @@ func newClientFromAlias(alias string, urlStr string) (Client, *probe.Error) {
 	// credentials from the match found in the config file.
 	s3Config := new(Config)
 	s3Config.AccessKey = hostCfg.AccessKey
-	s3Config.SecretKey = hostCfg.SecretKey
+
+	// secretKey retrieved from the environement overrides the one
+	// present in the config file
+	secretKeyEnv := os.Getenv("MC_SECRET_" + s3Config.AccessKey)
```

Comment:
> We can get both secret key and access key from the env. Can you please name them as 
> MC_SECRETKEY & MC_ACCESSKEY

The reason to use MC_SECRET_<ACESSKEY_ID> = SECRET
so that being able to support multiple access keys using just MC_SECRETKEY and MC_ACCESSKEY will not be able to provide such differentiation.